lib/fhir_models/r5/generated/resources/Account.rb

```diff
@@ -0,0 +1,210 @@
+module FHIR
+  module R5
+    class Account < Model
+      include FHIR::Hashable
+      include FHIR::Json
+      include FHIR::Xml
+
+      SEARCH_PARAMS = []
```

Comment:
This is populated for other versions.

Reply:
Fixed. The problem is that SearchParameter.xpath is deprecated in R5. This also fixed some missing search parameters in R4!